src/server/editionsPage.tsx

```diff
@@ -1,16 +1,16 @@
 // ----- Imports ----- //
 
-import { CacheProvider } from '@emotion/core';
+import { cache } from '@emotion/css';
```

Comment:
Why does the AR `page` use `createCache` but the ER `page` use this one? 🤔

Reply:
Because I forgot 🤦‍♂️ I'll update it 👍